packages/buckram/assets/styles/variables/_specials.scss

```diff
@@ -964,16 +964,54 @@ $examples-background: #f3e1e3 !default;
 /// @type String
 $examples-header-background: #7a333a !default;
 
+// Contributors
+
+/// Margin top for elements with a class of `contributors`.
+/// @type String
+$contributors-margin-top: 2rem !default;
+
+/// Margin bottom for elements with a class of `contributors`.
+/// @type String
+$contributors-margin-bottom: 2rem !default;
+
+/// Text indent for elements with a class of `contributors`.
+/// @type String
+$contributors-text-indent: 0 !default;
+
+/// Float property for contributor profile pictures.
+/// @param String
+$contributor-profile-picture-float: left !default;
+
+/// Padding for contributor profile pictures.
+/// @param String
+$contributor-profile-picture-margin: 0 1.5rem 0 0 !default;
```

Comment:
i was on the fence as to whether to create multiple margin-right, margin-left variables (in case a theme wanted to float images right, etc.) Open to suggestions.